Xin Wang added four functions on 2023/08/11.

Xin-666-hub opened the pull request:
Xin Wang added four functions on 2023/08/11. when users click theme change, it will be switch to night mode. when users click notification it will show notification on/off. when users click app update, it will show checking and the version is new one. when users click feedback it will show a form to users and when you type in something and click submit it will show feedback succeed.
Trello card: https://trello.com/c/1YRnAgAS/13-android-setting-page-function
test video: https://video.deakin.edu.au/media/t/1_197768ri

Xin-666-hub committed:
update setting page

Xin Wang added four functions on  2023/08/11. when users click theme change, it will be switch to night mode. when users click notification it will show notification on/off. when users click app update, it will show checking and the version is new one. when users click feedback it will show a form to users and when you type in something and click submit it will show feedback succeed.
Trello card: https://trello.com/c/1YRnAgAS/13-android-setting-page-function
test video: https://video.deakin.edu.au/media/t/1_197768ri

Xin-666-hub committed:
Merge branch 'Android_Development' of https://github.com/Gopher-Industries/Guardian into Android_Development

XinyaoCheng reviewed (changes requested):
Good work! Can you Darken the color of 4 buttons in seeting page a bit, after converting theme to dark

Xin-666-hub committed:
update setting page

Xin Wang added four functions on 2023/08/11. when users click theme change, it will be switch to night mode. when users click notification it will show notification on/off. when users click app update, it will show checking and the version is new one. when users click feedback it will show a form to users and when you type in something and click submit it will show feedback succeed.
Trello card: https://trello.com/c/1YRnAgAS/13-android-setting-page-function
test video: https://video.deakin.edu.au/media/t/1_2cqn5uhk
